lib/qa/authorities/linked_data/config.rb

```diff
@@ -18,10 +17,6 @@
 module Qa::Authorities
   module LinkedData
     class Config
-      class << self
```

Comment:
This should have been removed previously when the use of ERB was moved to another class.